ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -0,0 +1,611 @@
+              ParameterKey=InternalApiTargetGroupArn,ParameterValue="${INTERNAL_API_TARGET_GROUP}" \
+              ParameterKey=InternalServiceTargetGroupArn,ParameterValue="${INTERNAL_SERVICE_TARGET_GROUP}"
+          wait "$!"
+
+          aws cloudformation wait stack-create-complete --stack-name "${CLUSTER_NAME}-bootstrap" &
+          wait "$!"
+
+          BOOTSTRAP_JSON="$(aws cloudformation describe-stacks --stack-name "${CLUSTER_NAME}-bootstrap" \
+            --query 'Stacks[].Outputs[]' --output json)"
+
+          aws cloudformation create-stack \
+            --stack-name "${CLUSTER_NAME}-control-plane" \
+            --template-body "$(cat /var/lib/openshift-install/upi/aws/cloudformation/05_cluster_master_nodes.yaml)" \
+            --tags "${TAGS}" \
+            --parameters \
+              ParameterKey=ClusterName,ParameterValue="${CLUSTER_NAME}" \
+              ParameterKey=RhcosAmi,ParameterValue="${RHCOS_AMI}" \
+              ParameterKey=PrivateHostedZoneId,ParameterValue="${PRIVATE_HOSTED_ZONE}" \
+              ParameterKey=PrivateHostedZoneName,ParameterValue="${CLUSTER_NAME}.${BASE_DOMAIN}" \
+              ParameterKey=Master0Subnet,ParameterValue="${PRIVATE_SUBNET_0}" \
+              ParameterKey=Master1Subnet,ParameterValue="${PRIVATE_SUBNET_1}" \
+              ParameterKey=Master2Subnet,ParameterValue="${PRIVATE_SUBNET_2}" \
+              ParameterKey=MasterSecurityGroupId,ParameterValue="${MASTER_SECURITY_GROUP}" \
+              ParameterKey=IgnitionLocation,ParameterValue="https://api.${CLUSTER_NAME}.${BASE_DOMAIN}:22623/config/master" \
+              ParameterKey=CertificateAuthorities,ParameterValue="${IGNITION_CA}" \
+              ParameterKey=MasterInstanceProfileName,ParameterValue="${MASTER_INSTANCE_PROFILE}" \
+              ParameterKey=RegisterNlbIpTargetsLambdaArn,ParameterValue="${NLB_IP_TARGETS_LAMBDA}" \
+              ParameterKey=ExternalApiTargetGroupArn,ParameterValue="${EXTERNAL_API_TARGET_GROUP}" \
+              ParameterKey=InternalApiTargetGroupArn,ParameterValue="${INTERNAL_API_TARGET_GROUP}" \
+              ParameterKey=InternalServiceTargetGroupArn,ParameterValue="${INTERNAL_SERVICE_TARGET_GROUP}"
+          wait "$!"
+
+          aws cloudformation wait stack-create-complete --stack-name "${CLUSTER_NAME}-control-plane" &
+          wait "$!"
+        fi
+
+        openshift-install --dir=/tmp/artifacts/installer wait-for bootstrap-complete &
+        wait "$!"
+
+        # destroy bootstrap
+        if [[ "${CLUSTER_TYPE}" == "aws" ]]; then
+          aws s3 rm "s3://${BUCKET}" --recursive &
+          wait "$!"
+
+          aws s3api delete-bucket --bucket "${BUCKET}" &
+          wait "$!"
+
+          aws cloudformation delete-stack --stack-name "${CLUSTER_NAME}-bootstrap" &
+          wait "$!"
+
+          # TODO: 06_cluster_worker_node.yaml
+        fi
+
+        # TODO: e2e suite
```

Comment:
e2e suite is running in `test` container, it would start once setup is complete

Reply:
> e2e suite is running in `test` container...

:man_facepalming: Dropped in dac4d74a4 -> 007745d53.